pkgs/by-name/pr/proton-vpn-local-agent/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  python3,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "proton-vpn-local-agent";
+  version = "unstable-20241010";
```

Comment:
Ah, one last thing, please prefer the "standard" unstable version notation:

https://github.com/NixOS/nixpkgs/blob/6ae5c7581c172bf21b0998d3a6ad187d94b688bf/pkgs/README.md#L378

Reply:
So this would be: `0-unstable-2024-10-10`

Reply:
Done.